### ISSUE TITLE
Caching not working

### ISSUE THREAD

**Issue by wheredoesyourmindgo** [jameswyse/forecast#9]
Shouldn't Object.Assign be used to overwrite the default/static parameters with user supplied parameters?

I would think that the following... 

```javascript
this.options = Object.assign({}, options || {}, {
    service: 'darksky',
    units: 'celcius',
    cache: true,
    ttl: {
      minutes: 30
    }
  });
```

should be changed to this...

```javascript
this.options = Object.assign({}, {
    service: 'darksky',
    units: 'celcius',
    cache: true,
    ttl: {
      minutes: 30
    }
  }, options || {} );
```

**Comment by jameswyse** [comment]
Good catch, I'll fix this now :)

Where did my mind go indeed.. I should really write some unit tests

**Comment by jameswyse** [comment]
Should be fixed in 0.3.1. Thanks!
